content/docs/config-files.md

```diff
@@ -88,6 +88,34 @@ Each `<Item>` will go in the first level of the context menu. This can be overri
 
 Each `<Item>` will use the same text as the main menu entry uses, as defined by your current localization. This can be overridden by adding an `ItemNameAs="new_name_for_the_item"` attribute, so that the new name will be displayed instead of the standard one. This is useful when the name is lengthy, as it makes the Context Menu unwieldy otherwise. Non-Latin characters are supported.
 
+### Context menu syntax summary
+
+In brief, the structure of the context menu file is as follows
+
+- Top Level Node: `<NotepadPlus>` (no attributes)
+- Intermediate Node: `<ScintillaContextMenu>` (no attributes)
+- Entry Nodes: `<Item ...>`
+    - `MenuEntryName` attribute: Used for accessing Notepad++ menu items by name.  The value should be name of the menu.
+    - `MenuItemName` attribute: Used for accessing Notepad++ menu items by name.  The value should be the name of the menu item inside the given menu.
+    - `id` attribute: Used for accessing Notepad++ commands by CommandID.  The value should be the CommandID number, as seen in [english.xml](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/installer/nativeLang/english.xml) and similar translation files.
+        - `<Item id="0" />` will make a separator line in the menu.
+    - `ItemNameAs` attribute: Used for for [overriding a menu item name](#overriding-a-menu-item-name): using your own "display name" for a given entry in your context menu, instead of using the one inherited from the main Notepad++ menu system.
+    - `PluginEntryName` attribute: Used for accessing Plugin commands.  The value should be the name of the plugin as seen in the Plugins menu.
+    - `PluginCommandItemName` attribute: Used for accessing Plugin commands.  The value should be the name of the menu entry inside that Plugin's menu.

```

Comment:
Is it worth stating that said command must be in the first-level menu of the plugin, i.e., that commands that are "deeper than that" aren't available to be used?